News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/async_tasks/RssItemToHtmlTask.java

```diff
@@ -119,7 +119,7 @@ public static String getHtmlPage(RequestManager glide, RssItem rssItem, boolean
         String rtlDir = isRightToLeft ? "rtl" : "ltr";
 
         StringBuilder builder = new StringBuilder();
-        builder.append(String.format("<html dir=\"%s\"><head><meta name=\"viewport\" content=\"width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=0\" />", rtlDir));
+        builder.append(String.format("<html dir=\"%s\"><head><meta name=\"viewport\" content=\"width=device-width, initial-scale=1\" />", rtlDir));
```

Comment:
Unless the webview has zoom enabled the zoom restrictions here don't really matter anyway